### ISSUE TITLE
DotaTV can watch the tournament

### ISSUE THREAD

**Issue by LightFuryLF** [ValveSoftware/Dota2-Gameplay#11076]
### Description

it's happen again everytime you update something you broke something

### Screenshots

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/107495467/9ef13c3b-00c6-4454-ae62-72dfc772b861)

**Comment by shaileshgarg** [comment]
Dude please don't duplicate your own issues.

**Comment by LightFuryLF** [comment]
it's not my own issue it's community issue isn't? how they not care these problem make me want to spam it until they saw that they know the problem

imo I think they should do something that "OK we notice this problem" ok cool I just wait

but in this case is? idk they notice already or not

ok let's not fix it until TI yeah now is my own problem for sure

**Comment by T1HGO** [comment]
I agree bro, keep spamming. We have dreamleague in 2 days, and we won't be able to spectate in client. Less than one month is TI. Imagine not having this fixed by the time TI comes around, its just pathetic.

**Comment by LightFuryLF** [comment]
it's 23 days already is anybody here TI gonna start in 3 weeks

**Comment by T1HGO** [comment]
Valve busy listening to redditors crying about cosmetic bullshit.

**Comment by ragozh** [comment]
You have to join the match at the beginning, can't watch when the match already started
